src/schema/__tests__/sale_artwork.test.js

```diff
@@ -90,6 +90,35 @@ describe("SaleArtwork type", () => {
     })
   })
 
+  describe("artwork resolver", () => {
+    it("fetches artwork when edition_set data is requested", async () => {
+      const query = `
+        {
+          sale_artwork(id: "54c7ed2a7261692bfa910200") {
+            artwork{
+              id
+              edition_sets{
+                id
+              }
+            }
+          }
+        }
+      `
+      expect(await execute(query)).toEqual({
```

Comment:
in this test, can I check to see if the mocked `artworkLoader` above was used? we can kind of assume that based on the results but that's with assumption that original mocked artwork is not returning editionset.

Reply:
You can change the loader into a `jest.fn(...)` and store a reference to it and make assertions against it, e.g. `expect(loader).toHaveBeenCalledWith(...)`